docs/introduction/events.md

```diff
@@ -1,26 +1,29 @@
 # Events
 
-Events fall into two categories:
-- journal: tasks in the journal
-- project: tasks in other notes
+Events are any blocks in your database that have both a time dimension and a Logseq task property associated with them (e.g. TODO, DOING, LATER, NOW, WAITING)
```

Comment:
Or any block that is a child of the [[Daily Log]] block.  (I think)

Reply:
Blocks under the Daily Log are mainly used for interstitial journals. starting with version 2.5.0, they will be displayed separately on the new Daily Log page.